src/rebar3_hank.app.src

```diff
@@ -1,7 +1,7 @@
 {application,
  rebar3_hank,
  [{description, "A rebar plugin for dead code cleaning"},
-  {vsn, "1.1.1"},
+  {vsn, "1.1.2"},
```

Comment:
I would rather bump the version in a separate commit, since we will also need to adjust the changelog.